app/src/main/java/com/example/riden/ParseApplication.java

```diff
@@ -2,17 +2,22 @@
 
 import android.app.Application;
 
+import com.example.riden.models.Ride;
+import com.example.riden.models.User;
 import com.parse.Parse;
 import com.parse.ParseObject;
 
 public class ParseApplication extends Application {
+
     @Override
     public void onCreate() {
         super.onCreate();
+        ParseObject.registerSubclass(User.class);
+        ParseObject.registerSubclass(Ride.class);
 
         Parse.initialize(new Parse.Configuration.Builder(this)
-                .applicationId("eH5J7FhF6w6yIwFkuRgLs6r3IAUIKktKOk6qOOeK")
-                .clientKey("hAqK5klMLOr43tAbY3AWWh0gw8r3DAI0wF1F9xjL")
+                .applicationId("rNdnyJfDAU5iFJFmxM64xxJOoaY9VPZD4NInF0ie")
```

Comment:
Checking in the keys into a public repository may create problems for us. Anyone in the internet can use these. Let's chat about some alternatives